src/types/utility-types/recursive.ts

```diff
@@ -1,28 +0,0 @@
-import { IType, Type } from "../type"
```

Comment:
Yeah, only reason to keep that would be to provide a better describe() in circulars, but that's not bat at all as it works now.